ooiui/templates/science/streams.html

```diff
@@ -48,6 +48,11 @@
                 <div id="streamPanel"class="panel">
                     <div class="panel-heading">
                         <div class="pagination pagination-sm pagination-right pagination-boxes"></div>
+                        <div class="color-index">
+                            <div class="twelve-hours"></div><b> 12 Hours </b>
+                            <div class="twenty-four-hours"></div><b> 24 Hours </b>
+                            <div class="older"></div><b> Older than 24 hours </b>
+                        </div>
```

Comment:
@DanielJMaher totes missed it:
<img width="1432" alt="screen shot 2015-11-19 at 4 27 59 pm" src="https://cloud.githubusercontent.com/assets/3743882/11284922/9415c496-8eda-11e5-88dd-19f219bbb71c.png">